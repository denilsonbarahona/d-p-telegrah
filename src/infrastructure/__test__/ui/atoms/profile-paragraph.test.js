import React from "react";
import { render } from "@testing-library/react";
import ProfileParagraph from "@atoms/profileParagraph";

describe("Profile Paragraph", () => {
  it("to match snapshot", () => {
    const { container } = render(
      <ProfileParagraph>this is a test</ProfileParagraph>
    );
    expect(container).toMatchSnapshot();
  });
});
