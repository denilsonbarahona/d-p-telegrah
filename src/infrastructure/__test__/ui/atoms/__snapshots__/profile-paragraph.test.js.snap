// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Paragraph to match snapshot 1`] = `
<div>
  <p
    class="sc-bczRLJ gewZfU"
  >
    this is a test
  </p>
</div>
`;
